ghost-schema.js

```diff
@@ -1,133 +1,157 @@
-    featured: false,
-    page: true,
-    meta_title: 'Data schema',
-    meta_description: 'This is a data schema stub for Gatsby.js and is not used. It must exist for builds to function',
-    created_at: '2018-12-04T13:59:08.000+00:00',
-    updated_at: '2018-12-04T13:59:08.000+00:00',
-    published_at: '2018-12-04T13:59:14.000+00:00',
-    custom_excerpt: 'This is a data schema stub for Gatsby.js and is not used. It must exist for builds to function',
-    excerpt: 'This is a data schema stub for Gatsby.js and is not used. It must exist for builds to function',
-    codeinjection_head: '<style>.some-class {\n}<style><script>function(){console.log("hello");}</script>',
-    codeinjection_foot: '<style>.some-class {\n}<style><script>function(){console.log("hello");}</script>',
-    codeinjection_styles: '.some-class {\n}',
-    og_image: 'https://images.unsplash.com/photo-1532630571098-79a3d222b00d?ixlib=rb-0.3.5&q=80&fm=jpg&crop=entropy&cs=tinysrgb&w=1080&fit=max&ixid=eyJhcHBfaWQiOjExNzczfQ&s=a88235003c40468403f936719134519d',
-    og_title: 'Data schema',
-    og_description: 'This is a data schema stub for Gatsby.js and is not used. It must exist for builds to function',
-    twitter_image: 'https://images.unsplash.com/photo-1532630571098-79a3d222b00d?ixlib=rb-0.3.5&q=80&fm=jpg&crop=entropy&cs=tinysrgb&w=1080&fit=max&ixid=eyJhcHBfaWQiOjExNzczfQ&s=a88235003c40468403f936719134519d',
-    twitter_title: 'Data schema',
-    twitter_description: 'This is a data schema stub for Gatsby.js and is not used. It must exist for builds to function',
-    custom_template: 'post.hbs',
-    primary_author: author,
-    primary_tag: tag,
-    authors: [author],
-    tags: [tag],
-    visibility: 'public',
-    reading_time: 1
-};
+const types = `
+type GhostPost implements Node {
+    slug: String!
+    id: ID! 
+    uuid: String!
+    title: String!
+    html: String!
+    comment_id: String!
+    feature_image: String
+    featured: Boolean!
+    visibility: String!
+    created_at: String!
+    updated_at: String!
+    published_at: String!
+    custom_excerpt: String
+    codeinjection_head: String
+    codeinjection_foot: String
+    codeinjection_styles: String
+    custom_template: String
+    canonical_url: String
+    send_email_when_published: Boolean
+    tags: [GhostTag] @link(from: "tags.slug" by: "slug")
+    authors: [GhostAuthor]! @link(from: "authors.slug" by: "slug")
+    primary_author: GhostAuthor! @link(from: "primary_author.slug" by: "slug")
+    primary_tag: GhostTag @link(from: "primary_tag.slug" by: "slug")
+    url: String!
+    excerpt: String!
+    reading_time: Int!
+    og_image: String
+    og_title: String
+    og_description: String
+    twitter_image: String
+    twitter_title: String
+    twitter_description: String
+    meta_title: String
+    meta_description: String
+    email_subject: String
```

Comment:
```suggestion
    email_subject: String
    mobiledoc: String
    plaintext: String
    page: Boolean
```

This relates to comments about fields from "fake" post/page missing in GraphQL types - I don't know if this is correct - both of all those fields are actually there in GhostAPI response and if those should be nullable or not (so I didn't add `!` in types)

Reply:
On review, it looks like `page` is only set in the ghost pages API endpoint shape, as it is null in the ghost post endpoint shape. This means `page` should also be nullable.